Tools/TestWebKitAPI/Configurations/TestWGSL.xcconfig

```diff
@@ -0,0 +1,44 @@
+// Copyright (C) 2022 Apple Inc. All rights reserved.
+//
+// Redistribution and use in source and binary forms, with or without
+// modification, are permitted provided that the following conditions
+// are met:
+// 1. Redistributions of source code must retain the above copyright
+//    notice, this list of conditions and the following disclaimer.
+// 2. Redistributions in binary form must reproduce the above copyright
+//    notice, this list of conditions and the following disclaimer in the
+//    documentation and/or other materials provided with the distribution.
+//
+// THIS SOFTWARE IS PROVIDED BY APPLE INC. ``AS IS'' AND ANY
+// EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+// IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR
+// PURPOSE ARE DISCLAIMED.  IN NO EVENT SHALL APPLE INC. OR
+// CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL,
+// EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO,
+// PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR
+// PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY
+// OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+// (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+// OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+PRODUCT_NAME = TestWGSL;
+
+PROJECT_HEADER_SEARCH_PATHS = $(SRCROOT)/../../Source/WebGPU/WGSL $(SRCROOT)/../../Source/WebGPU/WGSL/AST $(SRCROOT)/../../Source/WebGPU/WGSL/AST/Expressions $(SRCROOT)/../../Source/WebGPU/WGSL/AST/Statements $(inherited);
+
+GCC_ENABLE_OBJC_EXCEPTIONS = YES;
+
+GCC_PREPROCESSOR_DEFINITIONS = $(inherited) BUILDING_TEST_WGSL GTEST_API_=
+
```

Comment:
In https://github.com/WebKit/WebKit/commit/a85b766e565b6a913b3fed1075d645934cd47cad, TestIPC was broken and I had to add 

```
// Workaround for "WebKit is not available..." error (rdar://91668054)
DISABLE_SDK_METADATA_PARSING[sdk=appletv*] = YES;
DISABLE_SDK_METADATA_PARSING[sdk=watch*] = YES;
```
to fix the tvOS and watchOS builds. Maybe we need the same here? I think the underlying Xcode issue was fixed in Xcode 14, so we might be safe now, but just calling it out in case you see any failures.